packages/lab/src/checkbox/Checkbox.css

```
@@ -1,43 +1,39 @@
 /* Styles applied to root component */
-.salt-density-medium {
-  --checkbox-fontSize: 12px;
-}
-.salt-density-touch {
-  --checkbox-fontSize: 16px;
-}
-.salt-density-low {
-  --checkbox-fontSize: 14px;
-}
-.salt-density-high {
-  --checkbox-fontSize: 11px;
-}
 
 .saltCheckbox {
-  --checkbox-label-marginTop: calc(var(--salt-size-unit) * var(--checkbox-top-multiplier));
-  --checkbox-space-multiplier: var(--saltCheckbox-space-multiplier, 3);
-  --checkbox-top-multiplier: var(--saltCheckbox-top-multiplier, 0.75);
+  --checkbox-gap-multiplier: var(--saltCheckbox-top-multiplier, 0.75);
 }
 
 .saltCheckbox {
-  line-height: var(--saltCheckbox-lineHeight, 1);
+  cursor: var(--salt-selectable-cursor-hover);
+  background: var(--saltCheckbox-background, none);
+  display: flex;
+  gap: calc(var(--salt-size-unit) * var(--checkbox-gap-multiplier));
```

```suggestion
  gap: var(--saltCheckbox-gap, calc(var(--salt-size-unit) * 0.75));
```

the multiplier token seems a bit unnecessary